portfolio2.css

```diff
@@ -0,0 +1,227 @@
+* {
+    box-sizing: border-box;
+	margin: 0;
+	padding:0;
+	font-family: ubuntu, sans-serif;
+}
+
+body {
+    margin: 0px;
+    /*background-image: url("https://images.pexels.com/photos/5044497/pexels-photo-5044497.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260");
+    background-color: #cccccc;*/
+
+   }
+	
+  div.links {
+		border-bottom-style: solid;
+		border-bottom-color: Lightgrey;
+		list-style: none;
+		margin: 0 2px;
+		padding: 10px;
+		display: flex;
+		justify-content: space-evenly;
+		transform: translate(0%, -1132%);
+		width:100%;
+		
+		
+	 }
+		
+	.links a{
+		display:inline;
+		justify-content: space-between;
+		color: #7393B3;
+		line-height:1.5em;
+		padding-left: 25px;
+		text-decoration:none;
+		height:90px;
+		font-size: 40px;
+		border-bottom-style: hidden;
+		
+	}
+	
+	.links a:hover{
+		border:2px #0096FF solid;
```

Comment:
going from 0px border to 2px border makes your elements move when hovered over. would have a 2px transparent border as part the styling as base to stop this happening